web/src/App.vue

```diff
@@ -16,11 +16,21 @@
     >
       {{ SERVER_DOWNTIME_MESSAGE }}
     </v-main>
+    <v-snackbar
+      v-model="showError"
+      :timeout="6000"
+      top
+      right
+      color="error"
+    >
+      Sorry, something went wrong on our side.
+      The developers have been notified, please try that operation again later.
```

Comment:
Just a small suggestion to avoid a run-on sentence.

```suggestion
      Sorry, something went wrong on our side
      (the developers have been notified). Please try that operation again later.
```